core/src/test/java/org/elasticsearch/search/aggregations/metrics/cardinality/InternalCardinalityTests.java

```diff
@@ -38,13 +37,21 @@
     private static List<HyperLogLogPlusPlus> algos;
     private static int p;
 
-    @Before
-    public void setup() {
+    @After //we force @After to have it run before ESTestCase#after otherwise it fails
```

Comment:
I don't understand why it is @After and not @Before anymore... This is going to break the test.